tests/test_write_json.py

```diff
@@ -39,90 +40,21 @@ def get_testdatatable():
         """
         Create a test dataframe
         """
-        data_table = pd.DataFrame()
-        data_table["TESTCAT"] = [-1, -1, 1, 2, 1, np.nan, 1, 2, 1, 2, 2, 1, 1, 2, 2]
-        data_table["TESTSTRING"] = [
-            "",
-            "a",
-            "b",
-            ".",
-            np.nan,
-            "c",
-            np.nan,
-            ".",
-            np.nan,
-            "d",
-            "e",
-            "f",
-            "f",
-            "f",
-            "g",
-        ]
-        data_table["TESTNUMBER"] = [-1, -1, -2, 5, 10, 10, 15, 100, 10, 2, -1, 3, 4, 5, 6]
-        data_table["TESTOTHER"] = [
-            -1,
-            "a",
-            -2,
-            5,
-            "b",
-            np.nan,
-            15,
-            "x",
-            2,
-            3,
-            "b",
-            1,
-            2,
-            "y",
-            "z",
-        ]
+        input_path = pathlib.Path("tests/input")
+        test_data = pd.read_csv(path.join(input_path, "test_data.csv"))
```

Comment:
Use input_path.join("test_data.csv") instead.

Reply:
input_path.joinpath("test_data.csv") works for pathlib